scrapers/scrapers/spiders/mo-bills.py

```diff
@@ -0,0 +1,687 @@
+import re
+import pytz
+import datetime as dt
+
+import requests
+
+from collections import defaultdict
+from io import BytesIO
+from zipfile import ZipFile
+
+from scrapy import Request, Selector, Spider
+
+from core.scrape import Bill, VoteEvent
+from ..items import Chamber
+
+bill_types = {
+    "HB ": "bill",
+    "HJR": "joint resolution",
+    "HCR": "concurrent resolution",
+    "SB ": "bill",
+    "SJR": "joint resolution",
+    "SCR": "concurrent resolution",
+}
+flags = [
+    ("Introduced", "introduction"),
+    ("Offered", "introduction"),
+    ("First Read", "reading-1"),
+    ("Read Second Time", "reading-2"),
+    ("Second Read", "reading-2"),
+    # make sure passage is checked before reading-3
+    ("Third Read and Passed", "passage"),
+    ("Reported Do Pass", "committee-passage"),
+    ("Voted Do Pass", "committee-passage"),
+    ("Third Read", "reading-3"),
+    ("Referred", "referral-committee"),
+    ("Withdrawn", "withdrawal"),
+    ("S adopted", "passage"),
+    ("Truly Agreed To and Finally Passed", "passage"),
+    ("Signed by Governor", "executive-signature"),
+    ("Approved by Governor", "executive-signature"),
+    ("Vetoed by Governor", "executive-veto"),
+    ("Vetoed in Part by Governor", "executive-veto-line-item"),
+    ("Legislature voted to override Governor's veto", "veto-override-passage"),
+]
+
+TIMEZONE = pytz.timezone("America/Chicago")
+
+custom_headers = {
+    "user-agent": "openstates.org"
+}
+
+
+class UnrecognizedSessionType(BaseException):
+    def __init__(self, session):
+        super().__init__(
+            f"Session {session} has the unrecognized session types."
+        )
+
+
+def lxmlize(url):
+    response = requests.get(url, headers=custom_headers)
+    return Selector(response)
+
+
+# remove whitespace, linebreaks, and end parentheses
+def clean_text(text):
```

Comment:
This looks genuinely like a general utility, so would be a good candidate for a utils.py file

Reply:
I've fixed it.